docker/web/Dockerfile

```diff
@@ -23,6 +23,7 @@ RUN set -x && apt-get update -qq \
     postgresql-server-dev-12 \
     default-jre \
     libgit2-dev \
+    libldap-2.4-2 \
```

Comment:
You add the LDAP dependency, but CodeCompass is built without the ldap auth plugin.

Reply:
Does this mean that I only need to add `-DWITH_AUTH="plain;ldap"` into the runtime container, since its content is copied in this Dockerfile?

Reply:
Yes.